docs/agents/sematext-agent/releasenotes.md

```diff
@@ -39,6 +39,23 @@ NULL
 
 -->
 
+## Version 3.2.2
+
+Date: January 8, 2024
+
+### New Features
+
+- Support has been added for [slog](https://go.dev/blog/slog) logs.
+
+### Improvements
+
+- Added `safe-to-evict` annotation to prevent downscaling `st-jmx` Kubernetes pods.
+- Added `org.elasticsearch.launcher.CliToolLauncher` signature to the Elasticsearch autodiscovery signatures.
+
+### Bug Fixes
+
+- Addressed a bug that was preventing the Sematext Agent from spawning monitoring app agent pods in Google Kubernetes Engine (GKE) environments.
```

Comment:
suggestion to rename: app agent -->
` [App Agent](https://sematext.com/docs/agents/sematext-agent/app-agent/)`